Neo4j.Driver/Neo4j.Driver/Internal/ConnectionPool.cs

```diff
@@ -34,6 +32,25 @@ internal static class PoolStatus
         public const int Active = 0;
         public const int Closed = 1;
         public const int Inactive = 2;
+
+        private static readonly IDictionary<int, string> StatusCodeToNameMappingTable = new Dictionary<int, string>
```

Comment:
The disadvantage of it is that we will have something like

```
sealed class PoolStatus {
public static PoolStatus Active = new PoolStatus(0, "Active");
public static PoolStatus Closed = new PoolStatus(1, "Closed");
public static PoolStatus Inactive = new PoolStatus(2, "Inactive");

public int Value;
public string Name;

private PoolStatus(int value, sting name)
{
  Value = value;
  Name = name;
}
```

Then in the code, when we need to refer to Active's value, then we need to use `PoolStatus.Value` which dose not looks good to read.
 For example it is a bit hard to read:

```Interlocked.Exchange(ref _poolStatus, Closed.Value) != Closed.Value```

Reply:
I think it is possible to use it like
```csharp
Interlocked.Exchange(ref _poolStatus, Closed) != Closed
```
If we have `_poolStatus` of type `PoolStatus`.

Not a big deal here, just thought it may look better.